components/session-service/oidc/oidc.go

```diff
@@ -91,9 +93,17 @@ func New(cfg Config, retrySec int, certs *certs.ServiceCerts, l OIDCLogger) (Cli
 		hc:       httpClient,
 		oauth2:   &oauth2cfg,
 		verifier: verifier,
-		log:      l}, nil
+		log:      l,
+		dexURL:   cfg.IssuerURL}, nil
 }
 
+func (c *client) RefreshTokenValidator(refreshToken string) (*http.Response, error) {
+	tokenValidatorURL := c.dexURL.String()
```

Comment:
tokenValidatorURL := c.dexURL.String() + "/refreshtokenvalid"

Reply:
instead of line 101 and 102, use fmt.Sprintf
tokenValidatorURL := fmt.Sprintf("%s/%s",c.dexURL.String(),"refreshtokenvalid")